tests/src/rules/no-commonjs.js

```diff
@@ -20,6 +20,7 @@ ruleTester.run('no-commonjs', require('rules/no-commonjs'), {
 
     // allowed requires
     { code: 'function a() { var x = require("y"); }' }, // nested requires allowed
+    { code: 'var a = c && require("b")' }, // dynamic requires allowed
```

Comment:
this is a conditional require, not a dynamic require.

Reply:
sorry for the ambiguity. i mean conditional require.